plugin-api.d.ts

```diff
@@ -1888,6 +1888,9 @@ interface BaseStyle extends PublishableMixin, PluginDataMixin {
   remove(): void
 }
 interface PaintStyle extends BaseStyle {
+  readonly boundVariables?: {
+    readonly paints: VariableAlias[];
+  };
```

Comment:
hm, I think that having `boundVariables` as a field on `paint` should be sufficient. styles contain an array of paints, each of which can be bound to a variable. styles themselves should not directly have bound variables